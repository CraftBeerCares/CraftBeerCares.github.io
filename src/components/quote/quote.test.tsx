import { render } from "@testing-library/react";
import Quote from "./quote";

describe("renders ArticleRow", () => {
  it.each`
    imageFirst
    ${true}
    ${false}
  `("imageFirst=$imageFirst", async ({ imageFirst }) => {
    const { container } = render(
      <Quote
        imageSrc="/assets/001.webp"
        imageAlt="test alt"
        imageFirst={imageFirst}
      >
        <p>Hello</p>
      </Quote>
    );
    expect(container).toMatchSnapshot();
  });
});
