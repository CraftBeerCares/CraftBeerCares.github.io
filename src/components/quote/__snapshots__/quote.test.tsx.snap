// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ArticleRow imageFirst=false 1`] = `
<div>
  <div
    class="quoteBox"
  >
    <img
      alt="Quote Start"
      class="quoteImage"
      data-nimg="1"
      decoding="async"
      height="24"
      loading="lazy"
      src="/assets/quote_start.svg"
      style="color: transparent;"
      width="24"
    />
    <p
      class="quote"
    />
    <img
      alt="Quote End"
      class="quoteImage"
      data-nimg="1"
      decoding="async"
      height="24"
      loading="lazy"
      src="/assets/quote_end.svg"
      style="color: transparent;"
      width="24"
    />
    <p
      class="author"
    />
  </div>
</div>
`;

exports[`renders ArticleRow imageFirst=true 1`] = `
<div>
  <div
    class="quoteBox"
  >
    <img
      alt="Quote Start"
      class="quoteImage"
      data-nimg="1"
      decoding="async"
      height="24"
      loading="lazy"
      src="/assets/quote_start.svg"
      style="color: transparent;"
      width="24"
    />
    <p
      class="quote"
    />
    <img
      alt="Quote End"
      class="quoteImage"
      data-nimg="1"
      decoding="async"
      height="24"
      loading="lazy"
      src="/assets/quote_end.svg"
      style="color: transparent;"
      width="24"
    />
    <p
      class="author"
    />
  </div>
</div>
`;
